lib/tasks/dss.rake

```diff
@@ -1,11 +1,20 @@
 require 'json'
 
 namespace :dss do
+  namespace :servers do
+    desc "Start solr and postgres servers using lando."
+    task :start do
```

Comment:
Thank you for adding this task!  We should update the README to use this instead of the old way to set up a dev environment.